scripts/r.fillnulls/r.fillnulls.py

```diff
@@ -481,7 +481,7 @@ def main():
                     tmp_rmaps.remove(holename + "_edges")
                     tmp_rmaps.remove(holename + "_dem")
                     tmp_vmaps.remove(holename)
-                except:
+                except Exception:
```

Comment:
This one shouldn't be that hard to figure out, `tmp_rmaps` is a list.

Reply:
Yeah, it was only using one method so thats why I just put general `Exception` for it. Otherwise it should only throw `ValueError`